browser/js/checkout/checkout.js

```diff
@@ -0,0 +1,28 @@
+//ui-view from incart
+//for now I just stuck it in home
+app.config(function ($stateProvider) {
+    $stateProvider.state("checkout", {
+        url: "/checkout",
+        templateUrl: "js/checkout/checkout.html",
+        controller: "checkoutCtrl",
+        resolve: {
+            user: function (AuthService, AddressFactory) {
+
+                return AuthService.getLoggedInUser()
+                    .then(function (user) {
+                        user.fullName = AddressFactory.getFullName(user.firstName, user.lastName)
```

Comment:
Can't we write a hook on the Model for the user's full name? Also, why did we choose to have a "name" function on the "address" Factory?